packages/editor/src/components/url-input/index.js

```diff
@@ -168,14 +168,14 @@ class URLInput extends Component {
 				if ( this.state.selectedSuggestion !== null ) {
 					event.stopPropagation();
 					const post = this.state.posts[ this.state.selectedSuggestion ];
-					this.selectLink( post.url );
+					this.selectLink( post.url, post );
 				}
 			}
 		}
 	}
 
-	selectLink( link ) {
-		this.props.onChange( link );
+	selectLink( link, post ) {
```

Comment:
Should we remove the first argument from this method, as it's already in `post.url`